generators/app/index.js

```diff
@@ -148,6 +148,12 @@ module.exports = generators.Base.extend({
         },
         {
           type: "confirm",
+          name: "pwa",
+          message: "Would you like to make a Progressive Web App?",
```

Comment:
do you want to ask this question again if user run `yo electrode` in an existing app?